lib/screens/login_screen.dart

```diff
@@ -92,63 +88,93 @@ class _LoginScreenState extends State<LoginScreen>
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(),
+    return PicosScreenFrame(
       body: Center(
-        child: Column(
-          children: <Widget>[
-            ElevatedButton(
-              onPressed: () {
-                _controller.forward();
-                // Navigator.of(context).pushNamed('/mainscreen');
-              },
-              child: const Text('log in'),
-            ),
-            SlideTransition(
-              position: _offsetAnimation,
-              child: FadeTransition(
-                opacity: _controller,
-                child: Column(
-                  mainAxisSize: MainAxisSize.max,
-                  children: <Widget>[
-                    SizedBox(
-                      width: 200,
-                      child: TextField(
-                        controller: _loginController,
-                        decoration: const InputDecoration(
-                          border: OutlineInputBorder(),
-                          labelText: 'Login',
-                        ),
-                      ),
+        child: PicosBody(
+          padding: 0,
+          child: Padding(
```

Comment:
Why do you use Padding, if you want to remove Padding from PicosBody?

Reply:
Valid Objection, redone!